server/tasks.py

```
@@ -161,6 +161,139 @@ def manage_alert_queue():
+        logger.info("Starting alert email processing job")
+
+        # Get pending or processing alerts for users with email enabled
+        pending_entries = (
+            db.query(AlertQueue)
+            .join(Alert, AlertQueue.alert_id == Alert.id)
+            .join(
+                UserAlertPreferences, AlertQueue.user_id == UserAlertPreferences.user_id
+            )
+            .filter(
+                AlertQueue.status.in_(["pending", "processing"]),
+                UserAlertPreferences.email_enabled == True,
```

Ruff (0.14.1)

Avoid equality comparisons to `True`; use `UserAlertPreferences.email_enabled:` for truth checks

Replace with `UserAlertPreferences.email_enabled`

(E712)